cob_bringup/package.xml

```diff
@@ -76,19 +76,20 @@
   <exec_depend>openni2_launch</exec_depend>
   <exec_depend>position_controllers</exec_depend>
   <exec_depend>realsense_camera</exec_depend>
+  <!--exec_depend>realsense2_camera</exec_depend--> <!-- not released into kinetic -->
   <exec_depend>robot_state_publisher</exec_depend>
   <exec_depend>rosserial_python</exec_depend>
   <exec_depend>rosserial_server</exec_depend>
   <exec_depend>rostopic</exec_depend>
   <exec_depend>rplidar_ros</exec_depend>
   <exec_depend>rviz</exec_depend>
-  <exec_depend>sick_visionary_t_driver</exec_depend>
+  <!--exec_depend>sick_visionary_t_driver</exec_depend-->
   <exec_depend>spacenav_node</exec_depend>
   <exec_depend>tf2_ros</exec_depend>
   <exec_depend>theora_image_transport</exec_depend> <!-- needed by the cameras to publish the compressed topics -->
   <exec_depend>topic_tools</exec_depend>
   <exec_depend>twist_mux</exec_depend>
-  <exec_depend>ur_driver</exec_depend>
+  <!--exec_depend>ur_driver</exec_depend-->
```

Comment:
`ur_driver` is not released into `melodic`...all robots using it, could stay on the `indigo_dev` branch and still use `kinetic`